src/components/mapgl/mapgl.component.ts

```diff
@@ -936,27 +948,27 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges {
     this.deleteSelectedItem();
   }
 
-  public onChangeStyle(styleGroupId: string, selectedStyleId: string) {
-    if (this.mapLayers && this.mapLayers.styleGroups) {
-      const selectedStyle: Style = this.getStyle(styleGroupId, selectedStyleId, this.mapLayers.styleGroups);
-      if (selectedStyle) {
-        this.mapLayers.styleGroups.filter(styleGroup => styleGroup.id === styleGroupId).forEach(styleGroup => {
-          styleGroup.selectedStyle = selectedStyle;
-        });
-        this.removeAllLayers();
-        if (selectedStyle.geomStrategy !== undefined) {
-          this.switchLayer.next(selectedStyle);
-        }
-        this.mapLayers.styleGroups.forEach(styleGroup => {
-          localStorage.setItem(this.LOCAL_STORAGE_STYLE_GROUP + styleGroup.id, styleGroup.selectedStyle.id);
-          styleGroup.selectedStyle.layerIds.forEach(layerId => {
-            this.addLayer(layerId);
-          });
-        });
-        this.onStyleChanged.next(this.mapLayers.styleGroups);
-      }
-    }
-  }
+  // public onChangeStyle(styleGroupId: string, selectedStyleId: string) {
```

Comment:
Do you need to keep the comments?

Reply:
For now it's kept, because I reuse some of the code blocks. I will remove it once it is completely replaced.